index.html

```diff
@@ -84,6 +84,254 @@ <h1 id="head-primary">
+
+          <div class="project-tags">
+            <ul>
+              <li class="html">HTML</li>
+              <li class="css">CSS</li>
+              <li class="javascript">JavaScript</li>
+            </ul>
+          </div>
+          <div class="project-action">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- third card -->
+      <div class="project-card third-card">
+        <div class="snapshoot">
+          <img src="./src/assets/img/project-3.svg" alt="third project snapshoot">
+        </div>
+
+        <div class="left-block">
+
+          <div class="project-title">
+            <div class="title">
+              <p>Tonic</p>
+            </div>
+            <div class="frame">
+              <ul>
+                <li class="client">CANOPY</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="role">Back End Dev</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="year">2022</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="project-info">
+            <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+
+          <div class="project-tags">
+            <ul>
+              <li class="html">HTML</li>
+              <li class="css">CSS</li>
+              <li class="javascript">JavaScript</li>
+            </ul>
+          </div>
+          <div class="project-action">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- fourth card -->
+      <div class="project-card fourth-card">
+        <div class="snapshoot">
+          <img src="./src/assets/img/project-4.svg" alt="fourth project snapshoot">
+        </div>
+
+        <div class="left-block">
+
+          <div class="project-title">
+            <div class="title">
+              <p>Multi-Post <br> Stories</p>
+            </div>
+            <div class="frame">
+              <ul>
+                <li class="client">CANOPY</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="role">Back End Dev</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="year">2022</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="project-info">
+            <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+
+          <div class="project-tags">
+            <ul>
+              <li class="tag html">HTML</li>
+              <li class="css">CSS</li>
+              <li class="javascript">JavaScript</li>
+            </ul>
+          </div>
+          <div class="project-action">
+            <button type="button">See Project</button>
```

Comment:
- Please kindly add the hover, enabled, disabled, and active effect on your buttons to match the design and to improve the user experience below:

![btnss](https://user-images.githubusercontent.com/59397159/178120612-7729def8-1181-4c73-ab60-76839298db47.png)